libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Extensions/PushNotificationManager.swift

```diff
@@ -91,6 +92,30 @@ extension PushNotificationManager {
 }
 
 extension PushNotificationManager {
+    @objc
+    func getNotificationTypes(restClient: RestClient = RestClient.shared,
```

Comment:
In the doc I was thinking the get* method returned cached data and there would be an internal fetch method which does the network call and is called during push notification registration or re-registration.

Since notificationTypes are exposed as a member of account we don't need a separate get method to get them. So your getNotificationTypes is really the method to fetch and store notification types / set categories. Maybe its name could reflect that? E.g. fetchAndStoreNotificationTypes.

Reply:
Yeah, I can make that change.